### PR TITLE
chore: Request package replace

### PR DESCRIPTION
**Pull request by FernandoAranda**
Issue #, if available:

Description of changes:
Replace request package with fetch 
Checklist:
- [x ] Have you successfully deployed to an AWS account with your changes?
- [x ] Have you successfully tested with your changes locally?
- [x] If new dependencies have been added, have they been pinned to specific versions?

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: addons/addon-base-rest-api/packages/services/package.json [hunk 1]
```diff
@@ -50,4 +50,4 @@
       "pre-commit": "pretty-quick --staged --pattern '**/*.*(js|jsx)'"
     }
   }
-}
+}
```
Comment (nguyen102) [inline addons/addon-base-rest-api/packages/services/package.json:53]: Nitpick: Add extra new lines in the files so GH doesn't complain

**Review by nguyen102 — COMMENTED** [review]
Looks good. Can you also update the threshold specified in this file?
https://github.com/awslabs/service-workbench-on-aws/blob/mainline/scripts/check-dependency-vulnerabilities.sh

File: addons/addon-base-rest-api/packages/services/package.json (hunk 1 above)
Comment (FernandoAranda) [reply addons/addon-base-rest-api/packages/services/package.json:53]: whenever I save the file with a new line it removes it, I will see what tool is doing that

**Comment by FernandoAranda** [comment]
> Looks good. Can you also update the threshold specified in this file? https://github.com/awslabs/service-workbench-on-aws/blob/mainline/scripts/check-dependency-vulnerabilities.sh

should I set it to 4/0 as it was before?

**Comment by FernandoAranda** [comment]
> Looks good. Can you also update the threshold specified in this file? https://github.com/awslabs/service-workbench-on-aws/blob/mainline/scripts/check-dependency-vulnerabilities.sh

Done!

**Review by nguyen102 — APPROVED** [review]

**Pull request merged**